src/dotnet/HQ.Server/Services/PlanService.cs

```diff
@@ -91,4 +97,20 @@ public PlanServiceV1(HQDbContext context)
             StaffId = previousPlan.StaffId
         };
     }
+    public async Task BackgroundSendPlanSubmissionReminderEmail(Period period, CancellationToken ct)
+    {
+        var date = DateOnly.FromDateTime(DateTime.UtcNow).GetPeriodStartDate(period);
+        var plans = _context.Plans.Where(t => t.Date == date);
+        var unsubmittedPlansStatus = plans.Where(t => t.Body == null || t.Status == null);
+
+        var staffToNotify = await _context.Staff
+            .AsNoTracking()
+            .Where(t => t.EndDate == null && plans.Where(x => x.StaffId == t.Id).Count() == 0 || unsubmittedPlansStatus.Where(x => x.StaffId == t.Id).Count() > 0)
+            .ToListAsync(ct);
+
+        foreach (var staff in staffToNotify)
+        {
+            _backgroundJobClient.Enqueue<EmailMessageService>(t => t.SendPlanSubmissionReminderEmail(staff.Id, date, CancellationToken.None));
+        }
+    }
```

Comment:
This needs to skip staff with any vacation or holiday time on the date being checked. Just check the Times table, by the time this runs the holiday time entries would have been auto generated and vacation would assumedly be entered by staff ahead of time so we can just check if they have any time for the vacation or holiday charge codes.